.circleci/config.yml

```diff
@@ -0,0 +1,939 @@
+          command: ./scripts/fix_error_ids.py --check-only
+
+  chk_pylint:
+    docker:
+      - image: buildpack-deps:eoan
+    steps:
+      - checkout
+      - run:
+          name: Install pip
+          command: apt -q update && apt install -y python3-pip
+      - run:
+          name: Install pylint
+          command: python3 -m pip install pylint z3-solver pygments-lexer-solidity
+          # also z3-solver to make sure pylint knows about this module, pygments-lexer-solidity for docs
+      - run:
+          name: Linting Python Scripts
+          command: ./scripts/pylint_all.py
+
+  chk_antlr_grammar:
+    docker:
+      - image: buildpack-deps:eoan
+    steps:
+      - checkout
+      - run:
+          name: Install Java
+          command: apt -q update && apt install -y openjdk-14-jdk
+      - run:
+          name: Run tests
+          command: ./scripts/test_antlr_grammar.sh
+
+  chk_buglist:
+    docker:
+      - image: circleci/node
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - run:
+          name: JS deps
+          command: |
+            npm install download
+            npm install JSONPath
+            npm install mktemp
+      - run:
+          name: Test buglist
+          command: ./test/buglistTests.js
+
+  chk_proofs:
+    docker:
+      - image: buildpack-deps:latest
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - run:
+          name: Z3 python deps
+          command: |
+            apt-get -qq update
+            apt-get -qy install python3-pip
+            pip3 install --user z3-solver
+      - run: *run_proofs
+
+  chk_docs_pragma_min_version:
+    docker:
+      - image: docker.pkg.github.com/ekpyron/solidity/solidity-buildpack-deps:ubuntu2004-1
+        auth:
+          username: docker
+          password: $GITHUB_TOKEN
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - run: *run_docs_pragma_min_version
+
+  b_ubu_clang: &build_ubuntu2004_clang
+    docker:
+      - image: ethereum/solidity-buildpack-deps:ubuntu2004-clang-<< pipeline.parameters.ubuntu-2004-clang-docker-image-rev >>
+    environment:
+      CC: clang
+      CXX: clang++
+    steps:
+      - checkout
+      - run: *run_build
+      - store_artifacts: *artifacts_solc
+      - persist_to_workspace: *artifacts_executables
+
+
+  b_ubu_asan_clang: &build_ubuntu2004_clang
+    docker:
+      - image: ethereum/solidity-buildpack-deps:ubuntu2004-clang-<< pipeline.parameters.ubuntu-2004-clang-docker-image-rev >>
+    environment:
+      CC: clang
+      CXX: clang++
+      CMAKE_OPTIONS: -DSANITIZE=address
+    steps:
+      - checkout
+      - run: *run_build
+      - store_artifacts: *artifacts_solc
+      - persist_to_workspace: *artifacts_executables
+
+  b_ubu: &build_ubuntu2004
+    docker:
+      - image: docker.pkg.github.com/ekpyron/solidity/solidity-buildpack-deps:ubuntu2004-1
+        auth:
+          username: docker
+          password: $GITHUB_TOKEN
```

Comment:
These are the only changes from the original config. They replace the docker image by the one in github packages. The username and password are a token generated for an *entirely unprivileged* github account, for which the token is *only* granted "read packages" permissions.
That token is named ``docker`` and the token itself is entered to circleci as a "secret" named ``GITHUB_TOKEN``.